app/javascript/react/components/molecules/StationValueLabel/StationValueLabel.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import BroadCastLogo from "../../../assets/icons/broadCastLogo.svg";
+import { LargeH1, H3 } from "../../../assets/styles/Typography";
+import * as S from "./StationValueLabel.style";
+
+interface StationValues {
+  date: string;
+  value: number;
+  parameter: string;
+}
+
+const StationValueLabel = ({ date, value, parameter }: StationValues) => {
+  return (
+    <S.GradientBox>
```

Comment:
Personally I'd go with a naming along those lines:
```
    <S.Container>
      <S.ImageContainer>
        <img src={BroadCastLogo} alt="Broadcasting logo" />
      </S.ImageContainer>
      <S.TextContainer>
        <H3>Avg for {date}</H3>
        <LargeH1>{value}</LargeH1>
        <H3>{parameter}</H3>
      </S.TextContainer>
    </S.Container>
```

I see this gradient as sth irrelevant form the html perspective and it is enough that it is define as a css property.